debug/main.py

```diff
@@ -71,7 +71,7 @@ def keepalives(prefix_or_iface):
         command += '"'
         run_command(command)
 
-    @bgp.command()
+    @bgp.command('neighbor_events')
```

Comment:
we should keep use - instead of underscore.

Reply:
Changed. Also changed in undebug/main.py.